src/main/java/user/delete_comment/DeleteCommentRequestModel.java

```diff
@@ -0,0 +1,37 @@
+package user.delete_comment;
+
+import entity.Comment;
+
+import java.util.ArrayList;
+
+public class DeleteCommentRequestModel {
+
+    private final ArrayList<Comment> comments;
```

Comment:
You don't need this variable. If it's parent is a `Comment`, you take in the parent's `commentId`. Since a `comment` could belong to a review OR a comment, you also need the `reviewId` so you can remove this comment from the reviews. Depending on your interactor you may need to decide if the parent is a `Comment` or a `Review`.